Solutions/SevcoSecurity/Data Connectors/Connector_SevcoSecurity.json

```diff
@@ -0,0 +1,95 @@
+{
+    "id": "SevcoDevices",
+    "title": "Sevco Platform - Devices",
+    "publisher": "Sevco Security",
+    "descriptionMarkdown": "The Sevco Platform - Devices connector allows you to easily connect your Sevco Device Assets with Microsoft Sentinel, to view dashboards, create custom alerts, and improve investigation. This gives you more insight into your organization’s assets and improves your security operation capabilities.\n\n[For more information >​](https://docs.sev.co/docs/microsoft-sentinel-inventory)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Sevco Devices CL",
+            "baseQuery": "Sevco_Devices_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description": "Summarize by local IP address",
+            "query": "Sevco_Devices_CL​ \n| summarize arg_max(TimeGenerated, *) by asset_id_g\n\n| extend IPs = parse_json(asset_attributes_ips_s), asset_attributes_hostname_s\n\n| mv-expand IPs\n\n| summarize count() by tostring(IPs)"
```

Comment:
Hello @MiahClark there is an extra whitespace after "sevco_devices_cl ,
can you please remove it